src/archanaNJun20/BookMeetingRoom.java

```diff
@@ -0,0 +1,105 @@
+package archanaNJun20;
+
+public class BookMeetingRoom {
+
+	String meetingRoom;
+	double hours=0.0;
+	boolean nalandaFlag=false;
+	boolean takshashilaFlag=false;
+	boolean agraFortFlag=false;
+	boolean pratapGadhFlag=false;
+
+	void bookMeetingRoom(){
+		if(nalandaFlag==false){
+			meetingRoom="Nalanda";
+			hours=1;
+			displayInfo();
+			nalandaFlag=true;
+		}
+		else{
+			System.out.println("-->The meeting room Nalanda is already booked.");
+		}
+	}
+	void bookMeetingRoom(String meetingRoom){
+		if(meetingRoom!="Nalanda"){
+			if(meetingRoom=="Takshashila"&&takshashilaFlag==false){
+				this.meetingRoom="Takshashila";
+				hours=1;
+				displayInfo();
+				takshashilaFlag=true;
+			}
+			else if(meetingRoom=="AgraFort" &&agraFortFlag==false){
+				this.meetingRoom="AgraFort";
+				hours=1;
+				displayInfo();
+				agraFortFlag=true;
+			}
+			else if(meetingRoom=="PratapGadh"&&pratapGadhFlag==false){
+				this.meetingRoom="PratapGadh";
+				hours=1;
+				displayInfo();
+				pratapGadhFlag=true;
+			}
+			else{
+				System.out.println(meetingRoom +"-->The meeting room Nalanda is already booked.");
+			}
+		}
+		else{
+			System.out.println(meetingRoom +"-->The meeting room Nalanda is already booked.");
+		}
+	}
+	void bookMeetingRoom(String meetingRoom,int hours){
+		if(meetingRoom!="Nalanda"){
+			if(meetingRoom=="Takshashila"&&takshashilaFlag==false){
+				this.meetingRoom="Takshashila";
+				this.hours=hours;
+				displayInfo();
+				takshashilaFlag=true;
+			}
+			else if(meetingRoom=="AgraFort"&&agraFortFlag==false){
+				this.meetingRoom="AgraFort";
+				this.hours=hours;
+				displayInfo();
+				agraFortFlag=true;
+			}
+			else if(meetingRoom=="PratapGadh"&&pratapGadhFlag==false){
+				this.meetingRoom="PratapGadh";
+				this.hours=hours;
+				displayInfo();
+				pratapGadhFlag=true;
+			}else{
+				System.out.println(meetingRoom+ "-->The meeting room  is already booked.");
+			}
+		}
+		else{
+			System.out.println("-->The meeting room Nalanda is already booked.");
+		}
+	}
+	void displayInfo(){
+		System.out.println( meetingRoom+" Meeting room "  + " is booked for " + hours + " hour");
+	}
+	void displayAvilableMeetingRoomForBooking() {
+		System.out.println("\nAvailable meeting Rooms:");
+		if(pratapGadhFlag==false){
+			System.out.println("*PratapGadh metting room is available*");
+		}
+		if(agraFortFlag==false)	{
+			System.out.println("*AgraFort metting room is available*");
+		}
+		if(nalandaFlag==false){
+			System.out.println("*Nalanda metting room is available*");
+		}
+		if(takshashilaFlag==false){
+			System.out.println("*Takshashila metting room is available*");
+		}
```

Comment:
in this you can add condition like what if all the meeting rooms are booked

Reply:
The code is updated as per your suggestion. Thank you.

Reply:
here you can add condition like if all the meeting rooms are booked and give warning to user if rooms are not available